augur/application/db/models/augur_data.py

```diff
@@ -66,10 +67,8 @@ class ChaossMetricStatus(Base):
 
     cms_id = Column(
         BigInteger,
+        Sequence('chaoss_metric_status_cms_id_seq', start=1),
```

Comment:
This needs `schema='augur_data'` - sequences don't inherit the table's schema automatically. Same applies to all other Sequence definitions in this file. See: https://docs.sqlalchemy.org/en/20/core/defaults.html#defining-sequences

```python
  Sequence('chaoss_metric_status_cms_id_seq', start=1, schema='augur_data')
```